9c-main/chart/templates/empty-chronicle.yaml

```
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: empty-chronicle
+  labels:
+    app: empty-chronicle
+    app.kubernetes.io/instance: {{ $.Chart.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: empty-chronicle
+  serviceName: empty-chronicle
+  template:
+    metadata:
+      labels:
+        app: empty-chronicle
+      name: empty-chronicle
+    spec:
+      initContainers:
+        - args:
+          - https://snapshots.nine-chronicles.com/main/partition
+          - /store
+          - $(RESET_SNAPSHOT_OPTION)
+          - empty-chronicle
+          - $(SLACK_WEBHOOK_URL)
+          command:
+          - /bin/download_snapshot.sh
+          image: {{ $.Values.emptyChronicle.image.repository }}:{{ $.Values.emptyChronicle.image.tag }}
+          name: reset-snapshot
+          volumeMounts:
+          - name: download-snapshot-script
+            mountPath: /bin/download_snapshot.sh
+            readOnly: true
+            subPath: download_snapshot.sh
+          - mountPath: /store
+            name: empty-chronicle-data
+          env:
+          - name: RESET_SNAPSHOT_OPTION
+            value: "false"
```

Since it is the first time to run this node, it should be `true` to run snapshot script.

```suggestion
            value: "true"
```

---

BTW, I'm looking into https://github.com/planetarium/9c-infra/issues/549 this issue and it hasn't been resolved yet so please change this back to "false" once you've finished downloading the snapshot 🙏